R/get-path-from-object.R

```diff
@@ -352,35 +311,6 @@ yaml_ext <- function(.x) {
 ###################################
 
 
-#' Build absolute path from a directory and path
```

Comment:
This is unrelated to the specific change, I just noticed that this isn't used anywhere and should've been deleted back when we did the `model_directory` refactor.